### PR TITLE
Fi 650 bulk data must support

### PR DESCRIPTION
**Pull request by czh-orz**
This adds must support testing capabilities to the bulk data test. It will loop through all the resources returned for a given resource and look for the associated must support elements. 

**Submitter:**
- [X] This pull request describes why these changes were made
- [X] Internal ticket for this PR: https://oncprojectracking.healthit.gov/support/browse/FI-650
- [X] Internal ticket links to this PR
- [X] Internal ticket is properly labeled (Community/Program)
- [X] Internal ticket has a justification for its Community/Program label
- [X] Code diff has been reviewed for extraneous/missing code
- [X] Tests are included and test edge cases
- [X] Tests/code quality metrics have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure
      where appropriate, and accomplishes the task's purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Comment by yunwwang** [comment]
I run a test with line_to_validate = 0, the result is not exactly correct. 
image

All validation tests should be skipped: 
image

**Comment by czh-orz** [comment]
> I run a test with line_to_validate = 0, the result is not exactly correct.
> image
> 
> All validation tests should be skipped:
> image

Should be fixed now

**Comment by yunwwang** [comment]
Find another issue when "validate all":
image

**Comment by czh-orz** [comment]
> Find another issue when "validate all":
> image

Woops, fixed.

File: lib/modules/onc_program/bulk_data_group_validation_sequence.rb [hunk 1]
```diff
@@ -230,7 +273,13 @@ def get_file(file, use_token: true)
           )
         end
 
-        test_output_against_profile('Patient')
+        must_supports = [
+          {
+            profile: nil,
+            must_support_info: Inferno::Sequence::USCore310PatientSequence::MUST_SUPPORTS.dup
```
Comment (arscan) [inline lib/modules/onc_program/bulk_data_group_validation_sequence.rb]: This depends on the fact that these files were loaded first.  Are we sure they are loaded first?

Comment (arscan) [reply lib/modules/onc_program/bulk_data_group_validation_sequence.rb]: @czh-orz @yunwwang I'm kicking this off into another ticket for us to consider how we really want these tests to be organized.  I'm concerned about mixing stuff across modules and the type of knowledge you would need to know about what depends on what, etc.  But I'd like to get this in to fill the MUST SUPPORT gap for now.
